resources/views/listing/partials/toolbar/stats.blade.php

```diff
@@ -1,11 +1,8 @@
 <ais-stats>
     <template v-slot="{ nbPages, nbHits, page, processingTimeMS }">
         @{{ nbHits }} @lang('products')
-        <template v-if="nbPages > 1">
-            (@lang('page'): @{{ page + 1 }} / @{{ nbPages }})
-        </template>
-        <template v-if="window.debug">
-            - @{{ processingTimeMS }}ms
-        </template>
+    </template>
+    <template v-if="window.debug">
```

Comment:
Does this work? As it's outside the `v-slot` where the `processingTimeMS` data is coming from.